.cirrus.yml

```diff
@@ -0,0 +1,53 @@
+env:
+  MAVEN_OPTS: -Xmx512M -XX:+ExitOnOutOfMemoryError
+  MAVEN_SKIP_CHECKS_AND_DOCS: -Dair.check.skip-all=true -Dmaven.javadoc.skip=true
+  MAVEN_FAST_INSTALL: -DskipTests $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T 1C
+
+webui_checks_task:
+  container:
+    image: node:latest
+  check_script: presto-main/bin/check_webui.sh
+
+maven_checks_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  test_script: mvn install -DskipTests -B -T 1C -P ci
+  docker_script: docker/build-local.sh
```

Comment:
How do they know `docker_script` is a name of executable and that it needs to run after `test_script`?

Reply:
See https://cirrus-ci.org/guide/writing-tasks/#script-instruction

It doesn't seem to explicitly say that they run in order, but it's clear from their examples that it works that way. We could send a PR to update the docs to say.